interfaces/PA-App/src/app/personal-components/enroll-in-program/enroll-in-program.component.ts

```diff
@@ -62,6 +62,7 @@ export class EnrollInProgramComponent extends PersonalComponent {
     this.programsService.getProgramById(programId).subscribe((response: Program) => {
       this.programTitle = this.mapLabelByLanguageCode(response.title);
       this.ngo = response.ngo;
+      this.paData.store('ngo', this.ngo);
```

Comment:
I think this doesn't really make sense.
The NGO that should be displayed is the one from "the program you're enrolling in", so therefore:
`this.paData.myPrograms[programId].ngo`. (it is already available)